common/featured_text.cpp

```diff
@@ -1208,3 +1207,21 @@ const char *unit_firepower_if_not_one(int firepower)
   }
   return buf;
 }
+
+/**
+   Get string of number of extra units killed by attacker
+   when a stack is taken down.
+   N.B.: The returned string is static, so every call to this function
+   overwrites the previous.
+ */
+const char *unit_n_stack_kills(int unitcount)
+{
+  static char buf[MAX_LEN_LINK];
+
+  if (unitcount > 2) {
+    fc_snprintf(buf, sizeof(buf), _("%d units were"), unitcount = 1);
+  } else {
+    fc_snprintf(buf, sizeof(buf), _("unit was"));
```

Comment:
You most likely do not want to implement [the rules for Arabic and French](https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html). Fortunately [the `PL_` macro](https://longturn.readthedocs.io/en/latest/Coding/internationalization.html) will do it for you.

Reply:
Happy to get rid of this if we have a way to _PL the line you mentioned later. My brain isn't running at very high capacity after all this cpp editing... :sweat_smile: